source/specifications.rst

```diff
@@ -23,13 +67,56 @@ other PyPA core reviewers, the lead PyPI maintainer and the lead
 CPython representative on distutils-sig, then they will have the
 authority to approve (or reject) that PEP.
 
-The current lead PyPI maintainer is Donald Stufft, while the current lead
-CPython representative on distutils-sig is Nick Coghlan.
+Otherwise, the default BDFL-Delegate depends on the area the PEP affects:
 
+* Package Distribution Metadata: lead CPython representative on distutils-sig
+* Package Index Interfaces: lead PyPI maintainer
 
-Active Specifications
----------------------
 
-The currently active specifications are recorded as a part of the
-:ref:`Python Packaging User Guide <pypug:specifications>`.
+Handling fixes and other minor updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The preferred approach to handling fixes and other minor updates adopted for
+all recent interoperability specifications is to submit an issue and/or pull
+request against the `official PEPs repo <https://github.com/python/peps>`_.
+
+All enhancements proposed this way *must* be discussed on distutils-sig prior
+to amending the PEP, and any changes made after PEP acceptance must be
+explicitly documented in a "Changes" section in the PEP itself. For example,
+see:
+
+* `Changes in PEP 440 <https://www.python.org/dev/peps/pep-0440/#summary-of-changes-to-pep-440>`_
+* `Changes in PEP 503 <https://www.python.org/dev/peps/pep-0503/#changes>`_
+
+PyPA core reviewers that are also PEP editos are responsible for deciding which
+of these changes can just be accepted (e.g. fixing a typo), which need to be
+reviewed by the relevant responsible decision maker before being accepted, and
+which need to be escalated to the full Python Enhancement Proposal process.
+
+One current exception to this policy is the core metadata specification in
+PEP 345, where additional fields and the selective superceding of various
```

Comment:
Might be reasonable to add ``and additional legacy psuedostandards`` or something along those lines.